### PR TITLE
(Feature) Changing of password

### PR DESCRIPTION
**Pull request by vbaranov**
Relates to https://github.com/poanetwork/metamask-extension/issues/75

The related issue to update e2e tests is created https://github.com/poanetwork/metamask-extension/issues/82

- `eth-keyring-controller` npm package is replaced with `eth-keychain-controller` to support change of password

### CODE REVIEW

File: old-ui/app/components/confirm-change-password.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+ConfirmChangePassword.prototype.ChangePassword = function () {
+  var oldPasswordBox = document.getElementById('old-password-box')
+  var oldPassword = oldPasswordBox.value
+  var newPasswordBox = document.getElementById('new-password-box')
+  var newPassword = newPasswordBox.value
+  var newPasswordConfirmBox = document.getElementById('password-box-confirm')
+  var newPasswordConfirm = newPasswordConfirmBox.value
+  if (oldPassword.length < 8) {
```
Comment (fvictorio) [inline old-ui/app/components/confirm-change-password.js]: If for some reason a user creates a password with less than 8 characters, this won't allow changing it. I know that passwords are validated to have that minimum length, but you never know. I would remove this unless there's some good reason to keep it.

File: old-ui/app/components/confirm-change-password.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+ConfirmChangePassword.prototype.createOnEnter = function (event) {
+  if (event.key === 'Enter') {
+    this.ChangePassword()
+  }
+}
+
+ConfirmChangePassword.prototype.ChangePassword = function () {
+  var oldPasswordBox = document.getElementById('old-password-box')
```
[the remark this answers is not in the document]
Comment (vbaranov) [reply old-ui/app/components/confirm-change-password.js]: agree, thanks, fixed

File: old-ui/app/components/confirm-change-password.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  var oldPassword = oldPasswordBox.value
+  var newPasswordBox = document.getElementById('new-password-box')
```
[the remark this answers is not in the document]
Comment (vbaranov) [reply old-ui/app/components/confirm-change-password.js]: Thanks for pointing me out. Refs were added for the part of the code, that is the subject of review. I guess, other bad patterns should be changed in the separate PRs.

File: old-ui/app/components/confirm-change-password.js (hunk 1 above)
Comment (vbaranov) [reply old-ui/app/components/confirm-change-password.js]: Ok, I skipped this validation here. Indeed, it looks redundant here.

**Pull request merged**